.github/workflows/reusable-run-integration-tests.yml

```diff
@@ -0,0 +1,129 @@
+---
+name: Reusable workflow to run integration tests with specific images
+on:
+  workflow_call:
+    inputs:
+      cloud_image_registry:
+        description: The cloud image registry to use.
+        type: string
+        required: false
+        default: ghcr.io
+      cloud_image_name:
+        description: The cloud image name to use.
+        type: string
+        required: false
+        default: calyptia/cloud/main
+      cloud_image_tag:
+        description: The image tag to use.
+        type: string
+        required: false
+        default: latest
+      target_repository:
+        description: Target repository where to find tests to run
+        type: string
+        required: false
+        default: ${{ github.repository }}
+      target_repository_ref:
+        description: Target repository ref where to find tests to run
+        type: string
+        required: false
+        default: ${{ github.ref }}
+      go_version:
+        description: Golang version to use for running tests
+        required: false
+        type: string
+        default: "1.17"
+      go_test_params:
+        description: Parameters to pass to go tests
+        required: false
+        type: string
+        default: "-v -race -tags=integration ./..."
+      enable_kind:
+        description: Enable a kind cluster.
+        required: false
+        type: boolean
+        default: false
+      auth0_client_id:
+        description: client used for OIDC authentication.
+        required: false
+        type: string
+    secrets:
+      fluentbit_config_validator_api_key:
+        description: Fluentbit config validator API key
+        required: false
+      fluentd_config_validator_api_key:
+        description: Fluentd config validator API Key
+        required: false
+      smtp_username:
+        description: SMTP username to test email notifications
+        required: false
+      smtp_password:
+        description: SMTP password to test email notifications
+        required: false
+      image_registry_username:
+        description: Password (if github use a PAT token) to use for authentication on private registry
+        required: false
+      image_registry_password:
+        description: Password (if github use a PAT token) to use for authentication on private registry
+        required: false
```

Comment:
I think all these secrets are required, what happens without them?

Reply:
added as required, you are correct.